src/Core/Migration/V6_6/Migration1739355802FixVatHandling.php

```diff
@@ -0,0 +1,48 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Migration\V6_6;
+
+use Doctrine\DBAL\ArrayParameterType;
+use Doctrine\DBAL\Connection;
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Migration\MigrationStep;
+
+/**
+ * @internal
+ *
+ * @codeCoverageIgnore
+ */
+#[Package('framework')]
+class Migration1739355802FixVatHandling extends MigrationStep
+{
+    /**
+     * @return array<string, string>
+     */
+    private const VAT_PATTERNS = [
+        'BE' => 'BE(\d{10}$|^0\d{9})',
```

Comment:
Maybe we shouldn't override custom patters and only update to the new ones if the old ones are in use? Like in `Migration1736154963FixCyprusVatIdPattern`? :thinking:

Reply:
![image](https://github.com/user-attachments/assets/51002ee2-7944-47ec-b23b-f7b93121b1e2)

Reply:
![Uploading the-sopranos-tony-soprano.gif…]()